platform_design/lib/tab_1.dart

```diff
@@ -0,0 +1,161 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+
+import 'tab_1_detail.dart';
+import 'utils.dart';
+import 'widgets.dart';
+
+const tab1Title = 'Songs';
+const tab1AndroidIcon = Icon(Icons.music_note);
+const tab1IosIcon = Icon(CupertinoIcons.music_note);
+const _itemsLength = 50;
+
+class Tab1 extends StatefulWidget {
+  const Tab1({ Key key, this.androidDrawerBuilder, this.platformOverride })
+      : super(key: key);
+
+  final WidgetBuilder androidDrawerBuilder;
+  final ValueNotifier<TargetPlatform> platformOverride;
```

Comment:
I don't have a strong opinion. You can decide. I was trying to avoid having any 'mechanisms' in this demo but the way it is now is somewhat mechanical too.